src/config/libinput.c

```diff
@@ -41,6 +42,7 @@ libinput_category_create(void)
 {
 	struct libinput_category *l = znew(*l);
 	libinput_category_init(l);
+	l->send_events_mode = LIBINPUT_CONFIG_SEND_EVENTS_ENABLED;
```

Comment:
This seems redundant as we call `libinput_category_init()` before this line which also sets the flag.

Reply:
Yeah, my bad :sweat_smile:. I just realised. Will update with relevant changes!